api/src/routes.ts

```diff
@@ -61,6 +61,10 @@ export const routes: IRoute[] = [
         path: "/stardust/block/metadata/:network/:blockId", method: "get",
         folder: "stardust/block/metadata", func: "get"
     },
+    {
+        path: "/stardust/milestone/latestmilestones/:network", method: "get",
+        folder: "stardust/milestone/latestmilestones", func: "get"
```

Comment:
The route and folder could be `stardust/milestone/latest`, as it's clear enough from the path the it's about milestones.